### DISCUSSION TITLE
README issues feedback

### DISCUSSION THREAD

**Discussion by MsJamie** [authelia/authelia#6857] [Documentation Feedback] [answered]
1. The README has several broken images 

![Screenshot_20240314_092330](https://github.com/authelia/authelia/assets/13186441/373cc531-8dd7-4f39-b1fb-99418409f451)

All those broken images the URLs return `Not Found`

https://camo.githubusercontent.com/9ce02bfa60d60ec648afa9db167bf737727322ad5f631cf16abb772301d991fb/68747470733a2f2f7777772e61757468656c69612e636f6d2f696d616765732f61757468656c69612d7469746c652e706e67

https://camo.githubusercontent.com/b6c5dfd6b2535158669d40e81c97646c5dc35b2a84afc561231fd23887ced393/68747470733a2f2f7777772e61757468656c69612e636f6d2f696d616765732f61726368692e706e67

https://camo.githubusercontent.com/eefd09b1a793ece9d148a1888ea3f63f53bdeac89845a39a3d79d35ce6ce33c5/68747470733a2f2f7777772e61757468656c69612e636f6d2f696d616765732f6c6f676f732f646f636b65722e6c6f676f2e706e67

https://camo.githubusercontent.com/b413f34e153ac1781459c32c51d8775ee8880cdb5a4e1487244eeeb85898b083/68747470733a2f2f7777772e61757468656c69612e636f6d2f696d616765732f3246412d4d4554484f44532e706e67

2. In the Security section. Both the `security` and `the documentation` links go to a 404 page on the main website

![Screenshot_20240314_092521](https://github.com/authelia/authelia/assets/13186441/c219146d-fd05-4fac-ae18-ec24cc1bb03c)

`security` > https://www.authelia.com/information/security/

`the documentation` > https://www.authelia.com/information/security/
This one is probably supposed to be https://www.authelia.com/overview/security/introduction/

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

To address your concerns:

1. For the broken images in the README, it seems like the URLs are pointing to resources that are either moved or deleted. The best course of action would be to report this issue directly to the Authelia team so they can update or fix the links to the images. You can do this by creating an issue on their GitHub repository or by contacting them through one of the methods listed on their contact page.

2. Regarding the broken links in the Security section leading to 404 pages, this also appears to be an issue with the website's content management. The correct action would be similar to the first point; you should report these broken links to the Authelia team. Given the context, it seems like the second link you mentioned (`the documentation`) was indeed supposed to lead to the security overview introduction based on the context provided. Reporting these will help improve the website for everyone.

For both issues, you can use the contact information provided on the Authelia website or directly create an issue on their GitHub repository. Make sure to provide them with detailed information about the broken images and links, including the URLs and where you found them.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by james-d-elliott** [comment, marked as answer]
Thanks for reporting, will take a look today now that the release is out of the way.
